SimulationAgent/DeviceState/DeviceStateActor.cs

```diff
@@ -25,13 +27,10 @@ public enum ActorStatus
             Updating
         }
 
-        public const string CALC_TELEMETRY = "CalculateRandomizedTelemetry";
+        public ISmartDictionary DeviceState { get; set; }
```

Comment:
Should we include the summary?
`     /// <summary> `
`        /// The virtual state of the simulated device. The state is`
`        /// periodically updated using an external script.`
`        /// </summary>`